docs/signer.md

```diff
@@ -0,0 +1,354 @@
+# Kritis Signer
+
+Kritis Signer is a command-line tool that creates attestations. These attestations can be used for Kritis and Binary Authorization deployment enforcement.
+
+The tool can create attestations directly or conditionally, checking policy conformance based on Vulnerability scanning results from Google Cloud Container Analysis.
```

Comment:
"The tool can create attestations directly or conditionally, checking policy conformance based on Vulnerability scanning results from Google Cloud Container Analysis." 

replace with:

Kritis Signer can also be used to create attestations based on vulnerability policy conformance. In this mode, the tool uploads your container image to [Google Container Registry]((https://cloud.google.com/container-registry) where [Google Container Analysis](https://cloud.google.com/container-registry/docs/container-analysis) performs [vulnerability scanning](https://cloud.google.com/container-registry/docs/vulnerability-scanning) and produces a vulnerability result for your container image. Kritis Signer then checks the vulnerability result against the vulnerability policy you configure.  If the vulnerability result complies with your policy, then Kritis Signer creates the attestation and stores it in the Container Analysis data store.

Reply:
Technically, the tool does not upload the image. I updated to, "your container image has to be uploaded to..."